examples/containers/docker-compose.yml

```diff
@@ -58,20 +55,23 @@ services:
             - 8080:8080
             # Cluster monitor
             - 7196:7196
+            # Loki
+            - 3100:3100
 
     # This section is intended to represent a typical local deployment, instead of containers it could be done with native binaries
     db1:
         container_name: db1
-        image: couchbase/server-sandbox:7.0.0
+        image: ${SERVER_IMAGE}
         ports:
             - 8091-8096:8091-8096
             - 11210-11211:11210-11211
         volumes:
+            - ./configure-cbs.sh:/entrypoint.sh
             - log-volume:/opt/couchbase/var/lib/couchbase/logs/:rw
         networks:
             - back
 
-    # Logging disabled
+    # Logging disabled if not using the custom entrypoint above.
     # permission-fixer:
```

Comment:
Do we need all this commented-out stuff now that we have the overridden entrypoint?

Reply:
Correctomundo so yeah I'll remove. Not sure about the exporter but that could be handled separately.